FSharpBuild.Directory.Build.props

```diff
@@ -11,7 +11,8 @@
   <PropertyGroup>
     <FSharpSourcesRoot>$(RepoRoot)src</FSharpSourcesRoot>
     <SymStoreDirectory>$(ArtifactsDir)\SymStore</SymStoreDirectory>
-    <ProtoOutputPath>$(ArtifactsDir)\Bootstrap</ProtoOutputPath>
+    <ProtoOutputPath Condition="'$(OS)' != 'Unix'">$(ArtifactsDir)\Bootstrap</ProtoOutputPath>
```

Comment:
@brettfo 
For windows we do the following, to decide whether to build the proto. in build.sh, it looks like we always build proto given that the uptodate check is pretty good, which is the right way to go?

````
function Update-Arguments() {
    if (-Not (Test-Path "$ArtifactsDir\Bootstrap\fsc.exe")) {
        $script:bootstrap = $True
    }
}

````